### PR TITLE
👋 add quickstart.py & more install instructions in readme

### PR DESCRIPTION
**Pull request by ezzcodeezzlife**
added written guide for beginner programmers and the quickstart code from readme as file.

Seems like many people want to use this Bot but are struggeling with the basic usage. ( #22, #20, #17)

This will help to get them started using this project.

@CamTosh thanks for the awesome project & please merge

### CODE REVIEW

**Comment by CamTosh** [comment]
Thank you for the contribution but it's just the code on readme, I don't think is really useful

**Comment by CamTosh** [comment]
Maybe add a simple script to download driver

**Comment by ezzcodeezzlife** [comment]
many want to use this but have really little to no knowledge in pyhton or programming. these install/download instructions  <b> will help them!</b> ❤️

**Comment by ezzcodeezzlife** [comment]
> Maybe add a simple script to download driver

done & ready to merge @CamTosh

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ if __name__ == '__main__':
 	insta.sendGroupMessage(users=['user1', 'user2'], message='Hey !')
 ```
 
+If you have little to no knowledge in programming: Read this Guide [How to use "instagram-bot-dm" Python for beginners](https://my-tailwind-nextjs-starter-blog.vercel.app/blog/how-to-use-instagram-bot-dm-python-script-for-beginners) 
+
```
Comment (CamTosh) [inline README.md:22]:
```suggestion

```

**Comment by CamTosh** [comment]
Quickstart is useless, the link to your blog too. Add a simple bash script / python package on lib for install chromedriver

**Pull request closed without merging**